File: problems/n/nsum/nsum.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    _ = int(input())

    numbers = [int(val) for val in input().split()]
    print(sum(numbers))

if __name__ == '__main__':
    main()

File: problems/n/nsum/nsum.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int n, number, total;

    cin >> n;

    total = 0;
    for (int i=0; i<n; i++) {
        cin >> number;
        total += number;
    }

    cout << total << endl;
}